Source/WebCore/platform/graphics/avfoundation/objc/VideoTrackPrivateAVFObjC.h

```diff
@@ -75,11 +75,11 @@ class VideoTrackPrivateAVFObjC final : public VideoTrackPrivateAVF {
     explicit VideoTrackPrivateAVFObjC(AVPlayerItemTrack*);
     explicit VideoTrackPrivateAVFObjC(AVAssetTrack*);
     explicit VideoTrackPrivateAVFObjC(MediaSelectionOptionAVFObjC&);
-    explicit VideoTrackPrivateAVFObjC(std::unique_ptr<AVTrackPrivateAVFObjCImpl>&&);
+    explicit VideoTrackPrivateAVFObjC(Ref<AVTrackPrivateAVFObjCImpl>&&);
 
     void resetPropertiesFromTrack();
     void videoTrackConfigurationChanged();
-    std::unique_ptr<AVTrackPrivateAVFObjCImpl> m_impl;
+    Ref<AVTrackPrivateAVFObjCImpl> m_impl;
```

Comment:
```suggestion
    const Ref<AVTrackPrivateAVFObjCImpl> m_impl;
```

will remove the CPP failures too.